CMakeLists.txt

```
@@ -127,7 +131,7 @@ else()
   set(PHASAR_STD_FILESYSTEM stdc++fs)
 endif()
 
-set(PHASAR_CONFIG_INSTALL_DIR "/opt/phasar/config/" CACHE STRING "Customizes the directory, where configuration files for PhASAR are installed (default is /opt/phasar/config)")
+set(PHASAR_CONFIG_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/.phasar-config/" CACHE STRING "Customizes the directory, where configuration files for PhASAR are installed (default is /usr/local/.phasar-config)")
```

```suggestion
set(PHASAR_CONFIG_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/.phasar-config/" CACHE STRING "Customizes the directory, where configuration files for PhASAR are installed (default is /usr/local/.phasar-config)" FORCE)
```
If we don't for the setting of the variable the cached state will stay the same, hence a change to `CMAKE_INSTALL_PREFIX` will not override the variable.